### PR TITLE
feature(Configuration reference / Content formatting options -> Local…

### PR DESCRIPTION
**Pull request by hamza0867**
…ization options): Update docs to new system

Related Ticket: TINY-8595

Description of Changes:
Update sections in Configuration reference: from Content formatting options to Localization options.
Worth noting some files are already fixed in #2328 ( namely: file_picker_callback.adoc )

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Comment by lnewson** [comment]
In `modules/ROOT/partials/configuration/images_file_types.adoc` there's a list that isn't rendering correctly. It needs to be changed to something like this:
```
This option configures which image file formats are accepted by the editor. Changing this option will adjust the following editor behaviour:

* Which image file formats are allowed to be uploaded in the xref:plugins/opensource/image.adoc[Image] dialog.
* Which image file formats are recognized and placed in an `img` element by the xref:plugins/opensource/paste.adoc[Paste] and xref:plugins/premium/powerpaste.adoc[PowerPaste] `smart_paste` functionality.
```

Additionally, where that is being included in `modules/ROOT/pages/configure/file-image-upload.adoc` the heading level is incorrect which is making it appear to be a subsection of `images_dataimg_filter`. As such we need to change the headings to use one less `=` and then update the include in `plugins/image.adoc` to use `leveloffset=+1`.

- [ ] Fix broken list
- [ ] Fix ToC/heading level issues

File: modules/ROOT/partials/configuration/content_langs.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ include::partial$misc/requires_5_9v.adoc[]
 
 include::partial$configuration/content_langs_base.adoc[]
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/content_langs.adoc:6]: The table in this file needs some formatting fixes, as most of the content is in the last column. I'd suggest adding `[cols="1,1,1,4"]` above the table (this is what we did in the 6.x docs).

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/configuration/content_langs.adoc (hunk 1 above)
Comment (hamza0867) [reply modules/ROOT/partials/configuration/content_langs.adoc:6]: Done

**Comment by hamza0867** [comment]
> In `modules/ROOT/partials/configuration/images_file_types.adoc` there's a list that isn't rendering correctly. It needs to be changed to something like this:
> 
> ```
> This option configures which image file formats are accepted by the editor. Changing this option will adjust the following editor behaviour:
> 
> * Which image file formats are allowed to be uploaded in the xref:plugins/opensource/image.adoc[Image] dialog.
> * Which image file formats are recognized and placed in an `img` element by the xref:plugins/opensource/paste.adoc[Paste] and xref:plugins/premium/powerpaste.adoc[PowerPaste] `smart_paste` functionality.
> ```
> 
> Additionally, where that is being included in `modules/ROOT/pages/configure/file-image-upload.adoc` the heading level is incorrect which is making it appear to be a subsection of `images_dataimg_filter`. As such we need to change the headings to use one less `=` and then update the include in `plugins/image.adoc` to use `leveloffset=+1`.
> 
> * [ ]  Fix broken list
> * [ ]  Fix ToC/heading level issues

Done

**Review by metricjs — APPROVED** [review]

**Pull request merged**